release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -0,0 +1,692 @@
+
+### Track PRs
+
+Track PRs based on the enhancement (KEP) spreadsheet.
+
+Keep the enhancement tracking spreadsheet up to date with review progress and merge status for each documentation PR. For example: [Kubernetes Enhancements OSS tracking board (1.14 release)](https://docs.google.com/spreadsheets/d/116X6E-lmDJG5UZPlqDAFw8hN9vS6SNY4qRNZ9fKtsMU/edit#gid=0)
+
+⚠️ Assign the enhancements evenly across your Shadows after the enhancement freeze. Assigning shadows before the
+ enhancement freeze will cause unexpected behavior in the tracking spreadsheet due to frequent updates to dynamic data.
+ (Tip: When assigning enhancements, try grouping enhancement owners and/or by SIGs)
+  - You and your Shadows will be responsible for tracking whether the enhancement:
+    - **has docs:** Actively has docs in place on Kubernetes website or in flight
+    - **needs docs:** Actively needs docs to support the enhancement in the [future-release]
+    - **unknown:** After reviewing the KEP, it is unclear if this needs docs or not
+    - **not required:** The KEP doesn't change anything that requires reflection in Kubernetes website
+  - You and your Shadows will also be responsible for marking whether the docs are:
+    - **Complete (Merged):** PR on the dev-[future-release] is done and merged
+    - **No PR:** No PR is open (helpful for dead line tracking)
+    - **Late:** No PR is open AND it's passed the deadline
+    - **Draft (PR):** PR is open but content isn't ready for review
+    - **Ready for Review (PR):** PR is open but content IS ready for review
+- ⚠️ Make sure that every docs PRs for the release have the correct base and set the correct Milestone. For example, enhancement PRs for version 1.14 need abase branch of `dev-1.14` and the Milestone set to `1.14`.
+
+The spreadsheet can be used to track the current health of the docs for release. For example:
+
+![Sample spreadsheet](pics/enhancement-tracking.png)
+
+On the "Enhancement Stats" tab, a table was created to track the Doc Stats and then based on their category, in the other tab, assign a status "Green / Yellow / Red."
+
+This is very helpful for weekly reports and managing deadlines
+
+### Reach out to Enhancement Owners
+
+#### Before the Open placeholder PR Deadline
+After the Enhancement Freeze, reach out to all tracked enhancements to see if new docs or modification to existing docs
+are required, e.g:
+
+> Hello < tag enhancement owner(s) > :wave:, < future-release > Docs < Shadow/Lead > here.
+>
+> Does this enhancement work planned for < future-release > require any new docs or modification to existing docs?
+>
+> If so, please follows the steps [here](https://kubernetes.io/docs/contribute/new-content/new-features/#open-a-placeholder-pr)
+> to open a PR against `k/website` repo. This PR can be just a placeholder at this time and must be created before < docs placeholder deadline >
```

Comment:
I know the [Open a Placeholder PR](https://kubernetes.io/docs/contribute/new-content/new-features/#open-a-placeholder-pr) instructions mentioned the `dev-branch` but how about aligning the language to open a PR against the `dev-1.xx` branch in the `k/website` repo on the role handbook

Reply:
Thanks for the feedback Rey! That's a great idea, it's been updated! :D